src/Phoenix.vue

```diff
@@ -7,8 +7,20 @@
       </template>
       <template v-else>
         <message-bar :active-messages="activeMessages" @deleteMessage="$_deleteMessage" />
-        <top-bar :applicationsList="$_applicationsList" :activeNotifications="activeNotifications" :user-id="user.id" :user-display-name="user.displayname" :hasAppNavigation="!!appNavigationEntries.length" @toggleAppNavigation="$_toggleAppNavigation(!appNavigationVisible)"></top-bar>
-        <side-menu :visible="appNavigationVisible" :entries="appNavigationEntries" @closed="$_toggleAppNavigation(false)"></side-menu>
+        <top-bar
+          :applicationsList="$_applicationsList"
+          :activeNotifications="activeNotifications"
+          :user-id="user.id"
+          :user-display-name="user.displayname"
+          :hasAppNavigation="appNavigationEntries.length > 1"
+          @toggleAppNavigation="$_toggleAppNavigation(!appNavigationVisible)"
```

Comment:
Sorry for being nitpicking here 😁 in my opinion, a function with name "toggle" should not have a parameter. I would either name it `$_setAppNavigationVisible` and provide a new value, or name it `$_toggleAppNavigationVisibility` and don't provide a parameter. You are legally allowed to hit me for that when we see each other.

Reply:
Implemented the "proper" toggle :wink: